demo/api/package.json

```diff
@@ -18,9 +19,9 @@
         "mikro-orm": "mikro-orm",
         "mikro-orm:drop": "mikro-orm schema:drop -r",
         "mikro-orm:migration:generate": "mikro-orm migration:create",
-        "start": "rimraf dist && dotenv -c -- nest start",
-        "start:debug": "rimraf dist && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
-        "start:dev": "rimraf dist && dotenv -c -- nest start --watch --preserveWatchOutput",
+        "start": "$npm_execpath prebuild && dotenv -c -- nest start",
+        "start:debug": "$npm_execpath prebuild && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
+        "start:dev": "$npm_execpath prebuild && $npm_execpath db:migrate && $npm_execpath console createBlockIndexViews && dotenv -c -- nest start --watch --preserveWatchOutput",
```

Comment:
Please don't more than needed in one PR. Here you move the call to db:migrate from dev-pm.config into api/package.json. If you think that should be changed, do that in a second PR.

Reply:
We already made that change in the starter and many of our projects. This PR retroactively adds those changes to Demo. 

If you have a strong concern regarding this, we can discuss it. Otherwise I see no point in splitting the PR because it causes overhead for everybody.